### DISCUSSION TITLE
Extract Audio Track from cut segment/s via Right Click menu or main export settings

### DISCUSSION THREAD

**Discussion by isobront232** [mifi/lossless-cut#1917] [Ideas]
It would be great if within the right click menu of the selected track segment, there was an option to export the audio track combined or as separate audio tracks.

Additionally, within the main export options window,  having the audio only as an export option (combined or separate tracks) would bring added benefits.

In my situation having an audio only version in addition to the cut video stream is a necessary part of my workflow. Being able to export the video streams separately along with the combined or separate audio tracks in one or two clicks would be a great feature. For windows the application Bandicut has a similar export feature set for reference. 

Thanks for the amazing work and contribution to the community.

**Comment by mifi** [comment]
> It would be great if within the right click menu of the selected track segment, there was an option to export the audio track combined or as separate audio tracks.

currently I think you can achieve this by:
1. right clicking a segment and then clicking "select only this segment" (or use the existing `selectOnlyCurrentSegment` keyboard action)
2. open the Tracks panel
3. "discard all tracks"
4. if you want to export each audio track as separate files
  - enable only the first audio track, then export
  - enable only the second audio track, then export
5. if you want to export all audio track as one file
  - click `...` next to an audio track, "Keep all audio tracks"
  - export

I think your workflow sounds like a bit niche, so I'm not sure if it makes sense to implement this specific feature which adds a lot of complexity, however I think with some keyboard shortcuts it could be possible to improve your workflow (and others) to a few keypresses only:
- new keyboard shortcuts for the following actions:
  - disable all tracks
  - enable all audio tracks
  - enable all video tracks
  - enable audio track number #N
  - enable video track number #N

> Additionally, within the main export options window, having the audio only as an export option (combined or separate tracks) would bring added benefits.

with the above keyboard shortcuts i think this would not be necessary.

let me know what you think.

added here: #254
